Source/src_android/PinCheckerLibrary/src/main/java/com/wultra/android/passphrasemeter/PasswordTester.java

```diff
@@ -90,13 +92,17 @@ private PasswordTester() {}
      * Tests the strength of the password. Before you test the passwords, you should load a right
      * dictionary by calling {@link #loadDictionary(AssetManager, String)} method.
      *
-     * @param password Password to test
+     * @param passwordBytes Password to scan as byte array.
      * @return Strength of the password
      * @throws WrongPasswordException if provided password is {@code null} or cannot be converted
      *         to C-string.
      */
-    public PasswordStrength testPassword(@NonNull String password) throws WrongPasswordException {
-        switch (testPasswordJNI(password)) {
+    public PasswordStrength testPassword(@NonNull byte[] passwordBytes) throws WrongPasswordException {
+
+        int bufferLength = passwordBytes.length;
+        ByteBuffer buffer = getSharedBuffer(passwordBytes, bufferLength);
```

Comment:
The `length` is a part of byte array, so there's no need to introduce a special length parameter here. Basically, by passing array itself we also pass the length of the array to the function.